cmd/serve/serve.go

```diff
@@ -296,6 +292,11 @@ func Serve(log *zap.Logger, listenClientUrls []string) error {
 	setupSigHandler()
 	fmt.Printf("Press Ctrl-C to quit\n")
 
+	if !s.UseMonitorSocket() {
+		// Don't connect to monitor socket. Block here.
+		select {}
```

Comment:
🤔 

Trying to decide if this signals something bad about this function that it requires noop blocking

Reply:
we probably don't need to do this. on hubble side i'm doing https://github.com/cilium/cilium/commit/f0db2ca2e9ef82fb01e1150b2aca4dd2e87274ea#diff-1113406423fe93e2d600aa0f9e69183bR1353

we can call runHubble without creating a new goroutine if Serve() doesn't block i think.

Reply:
something like this: https://github.com/cilium/cilium/commit/0c0f38988608d10706dfa9b8e19895c2bcbe4125#diff-1113406423fe93e2d600aa0f9e69183bR1352